.cbmc-batch/jobs/aws_cryptosdk_decrypt_body/aws_cryptosdk_decrypt_body_harness.c

```diff
@@ -41,7 +41,9 @@ void aws_cryptosdk_decrypt_body_harness() {
 
     struct content_key *content_key;
 
+    /* Need to allocate tag_len bytes for writing the tag */
     uint8_t *tag = can_fail_malloc(props->tag_len);
+    __CPROVER_assume(tag != NULL);
```

Comment:
is this a precondition on the function

Reply:
It wasn't but I added the more refined and required AWS_MEM_IS_WRITABLE as a check in a previous commit.

Reply:
Good